packages/esm-service-queues-app/src/queue-table/default-queue-table.component.tsx

```diff
@@ -135,14 +135,15 @@ function DefaultQueueTable() {
               ExpandedRow={QueueTableExpandedRow}
               tableFilters={
                 <>
-                  <QueueDropdownFilter /> <StatusDropdownFilter />
+                  <ClearQueueEntries queueEntries={filteredQueueEntries} />
```

Comment:
The mockup does not match the requirements. We need both a dropdown for filtering by service (queue) and a dropdown for filtering by status.

Reply:
Sorry, I just saw Ciaran's comment in [the ticket](https://openmrs.atlassian.net/browse/O3-4119) about moving the QueueDropdownFilter into the header. We might want to do that in the same PR; otherwise the change will break functionality.

Reply:
@chibongho so in the meantime we wait for the ticket to be created by Ciaran?